guides/common/modules/proc_adding-installation-media-by-using-cli.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-installation-media-by-using-cli"]
+= Adding installation media to {Project} by using Hammer CLI
+
+[role="_abstract"]
+Installation media are sources of packages that {ProjectServer} uses to install a base operating system on a machine from an external repository.
+ifdef::foreman-el,katello[]
+When you install the Katello plugin, you can download packages from a Pulp mirror.
+In this case, installation media are ignored.
+endif::[]
+ifdef::satellite[]
+You can use this parameter to install third-party content.
+Red Hat content is delivered through repository syncing instead.
```

Comment:
```suggestion
Red{nbsp}Hat content is delivered through synchronized Kickstart repositories instead.
```

Or more specifically:

Content to provision {RHEL} hosts is delivered through Kickstart repositories.

Reply:
Yeah that works.